BusinessLogicLayer/DTOs/RegisterDTO.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogicLayer.DTOs
+{
+    public class RegisterDTO
+    {
+        public string Username { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string ConfirmPassword { get; set; }
```

Comment:
We don't need to check it on server, we can check in on client

```suggestion
        public string Password { get; set; }
```